src/app/screens/Keysend/index.tsx

```diff
@@ -7,29 +7,20 @@ import SatButtons from "@components/SatButtons";
 import SuccessMessage from "@components/SuccessMessage";
```

Comment:
This shouldn't be in this PR, right?
It's covered in #1488 I think

Reply:
I didn't checkout from master it seems :P